cmd/istioctl/main.go

```diff
@@ -163,11 +178,11 @@ var (
 					c.Println(c.UsageString())
 					return err
 				}
-				item, exists := config.Get(key)
-				if !exists {
-					return fmt.Errorf("%q does not exist", key)
+				config, err := apiClient.GetConfig(key)
+				if err != nil {
+					return err
 				}
-				out, err := schema.ToYAML(item)
+				out, err := schema.ToYAML(config.ParsedSpec)
```

Comment:
I'm still not sure why we need schema here. The API server can just send proto-decoded JSON and we simply convert JSON to YAML, right?